python/tank/deploy/tank_commands/move_pc.py

```diff
@@ -97,7 +97,12 @@ def _copy_folder(self, log, level, src, dst):
     
     
     def run_interactive(self, log, args):
-        
+
+        if self.tk.pipeline_configuration.is_unmanaged():
```

Comment:
Instead of having to update all commands to check for the flag, maybe there should be a property on each command that says if the can operate of managed or unmanaged and depending on the config we are using they show up or not?